### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by zmxhawrhbg**
fix typo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@
 **GDDocs** is a project built to openly give advanced information and readable information for aspiring developers looking to interface with Geometry Dash. Primarily, we aim to create this as a website for people to learn more about the inner workings of geometry dash, along with it's data.
 
 ## Running/Building
-**You will require Node.js `>12` or above to debug and run this project.**
```
Comment (Homurasama) [inline README.md:15]: `12<` *

Comment (zmxhawrhbg) [reply README.md:15]: 12 or above or above

Comment (Homurasama) [reply README.md:15]: remove the `or above`

**Comment by Homurasama** [comment]
thank u

**Comment by SMJSGaming** [comment]
this was apparently not merged yet so lemme do that

**Pull request merged**